src/app/pages/auth/components/token-form/token-form.component.html

```diff
@@ -1,4 +1,4 @@
-<form [formGroup]="metaQRForm" id="tokenForm" (ngSubmit)="submitForm()">
```

Comment:
I added the `id` here because it is being used to link the submit button `<button form='tokenForm'>` here: https://github.com/RADAR-base/RADAR-Questionnaire/blob/ca3d6c7544358db6e1c3396cb49891447397932b/src/app/pages/auth/containers/enrolment-page.component.html#L108-L118